macros/datetime/date_spine.sql

```diff
@@ -44,13 +44,13 @@ with rawdata as (
 all_periods as (
 
     select (
-        {{
+        cast({{
             dbt_utils.dateadd(
                 datepart,
                 "row_number() over (order by 1) - 1",
                 start_date
             )
-        }}
+        }} as date)
```

Comment:
if we cast this to a date, we'll drop the "time" component. Can you make this work by casting as a timestamp instead of a `date`?

Reply:
see above lol